pangea-sdk/v3/internal/pangeatesting/integration.go

```diff
@@ -40,9 +40,10 @@ func IntegrationAuditVaultConfig(t *testing.T, env TestEnvironment) *pangea.Conf
 
 func IntegrationCustomSchemaConfig(t *testing.T, env TestEnvironment) *pangea.Config {
```

Comment:
This is just for custom schema, have you updated the other ones? Or it wasn't needed?

Reply:
The other search tests use `IntegrationConfig()` above which already has it.